src/openms/include/OpenMS/FILTERING/TRANSFORMERS/SpectraMerger.h

```diff
@@ -297,22 +298,76 @@ namespace OpenMS
       exp.sortSpectra();
     }
 
+    /**
+     * @brief check if the first and second mzs might be from the same mass
+     *
+     * @param mz1 the first m/z value
+     * @param mz2 the second m/z value
+     * @param tol_ppm tolerance in ppm
+     * @param max_c maximum possible charge value
+     */
+    bool areMassesMatched(double mz1, double mz2, double tol_ppm, int max_c)
```

Comment:
inputs should be `const`, just for good measure?!

Reply:
Oh you are right.. Thanks. Should be const.

Reply:
`max_c` should be uint, or you should internally test for it being >=1, because otherwise the algorithm will always return false.